mobile/src/main/java/com/truckmuncher/app/vendor/settings/VendorSettingsFragment.java

```diff
@@ -0,0 +1,32 @@
+package com.truckmuncher.app.vendor.settings;
+
+
+import android.os.Bundle;
+import android.preference.PreferenceFragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.truckmuncher.app.R;
+
+public class VendorSettingsFragment extends PreferenceFragment {
+
+    public static VendorSettingsFragment newInstance() {
+        return new VendorSettingsFragment();
+    }
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        addPreferencesFromResource(R.xml.vendor_preferences);
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
```

Comment:
Since it's a `PreferenceFragment`, don't set a layout. That happens automatically b/c in the preferences xml you set the parent element to be a `PreferenceScreen`